tests/lib/rules/compound-button-needs-labelling-v9.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/compound-button-needs-labelling-v9"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+ruleTester.run("compound-button-needs-labelling-v9", rule, {
+    valid: [
+        '<CompoundButton aria-label="Compound Example" />',
```

Comment:
We can add the icon prop to the tests to have a more realistic test case.

e.g.

correct

```

  <CompoundButton>
    Example
  </CompoundButton>
  
  <CompoundButton secondaryContent="Secondary content">
    Example
  </CompoundButton>
  
   <CompoundButton
    icon={<CalendarMonthRegular />}
    secondaryContent="Secondary content"
  >
    Example
  </CompoundButton>
   
       <Tooltip content="With calendar icon only" relationship="label">
        <CompoundButton icon={<CalendarMonthRegular />} />
      </Tooltip>
	  
  <CompoundButton
    icon={<CalendarMonthRegular />}
    aria-label="Monthly calendar"
  >
  </CompoundButton>
	  
  
  <CompoundButton
    icon={<CalendarMonthRegular />}
    secondaryContent="Secondary content"
  >
  </CompoundButton>
  
  
  <CompoundButton
    icon={<CalendarMonthRegular />}
    secondaryContent="Secondary content"
  >
  </CompoundButton>
  
  
    <Label id="calendar-1">Compound example</Label>
    <CompoundButton aria-labelledby="calendar-1" icon={<CalendarMonthRegular />} />

```
  
  incorrect
  
```
    <CompoundButton></CompoundButton>
  
  
    <CompoundButton icon={<CalendarMonthRegular />}></CompoundButton>
	
	
	<Label>Compound example</Label>
    <CompoundButton icon={<CalendarMonthRegular />} />
```